starknet_py/contract.py

```diff
@@ -45,14 +52,23 @@
 class ContractData:
     address: int
     abi: ABI
-    identifier_manager: IdentifierManager
+
+    @cached_property
+    def identifier_manager(self) -> IdentifierManager:
+        warnings.warn(
+            "Using identifier_manager from ContractData has been deprecated. Consider using parsed_abi instead."
+        )
+        return identifier_manager_from_abi(self.abi)
+
+    @cached_property
+    def parsed_abi(self) -> Abi:
+        return AbiParser(self.abi).parse()
```

Comment:
Is it documented somewhere?

Reply:
Maybe a docstring would be good here?

Reply:
This is basically a property of the object, but since it is frozen it is easier to do it this way.

Reply:
I can add docstring